runner.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/bash
+#!/usr/bin/python3
+module load nextflow-20.10.0
+module load singularity-3.7.1
+
+INPUT_DIRS=data/
+OUT_DIRS=$(cd $INPUT_DIRS; find * -type d -prune -exec ls -d {} \; |head -1)
+echo $OUT_DIRS
+
+ILLUM_DIRS=data/illumina/
+ILLUM_RES=$(cd $ILLUM_DIRS; ls -td * |head -1)
+
+ARTIC_DIRS=data/artic/
+ARTIC_RES=$(cd $ARTIC_DIRS; find * -type d -prune -exec ls -d {} \; |head -1)
+
+MIDNIGHT_DIRS=data/midnight/
+MIDNIGHT_RES=$(cd $MIDNIGHT_DIRS; find * -type d -prune -exec ls -d {} \; |head -1)
```

Comment:
Multiple hardcoded paths. Also no checking for existence and failsafes.

Reply:
True. Will try to find more suitable way.

Reply:
removed from the main script.